doc/introduction/quantum_datasets.rst

```diff
@@ -0,0 +1,9 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_qdata:
+
+Quantum Datasets
+=================
+
+Placeholder
```

Comment:
Does this need to be removed?

Reply:
Yes, it is removed [here](https://github.com/PennyLaneAI/pennylane/pull/3187).